Dockerfile

```diff
@@ -1,18 +1,24 @@
 ARG NGX_VERSION=1.28.0
-ARG NGX_DEBUG=false
+ARG NGX_CONFIGURE_ARGS=
```

Comment:
It seems NGX_CONFIGURE_ARGS is only used in `cargo build`, so it is not needed in the global ARG scope.

```diff
diff --git a/Dockerfile b/Dockerfile
index e5548275..770ba067 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -1,5 +1,4 @@
 ARG NGX_VERSION=1.28.0
-ARG NGX_CONFIGURE_ARGS=
 # NGINX official images are available for one specific release of Debian,
 # e.g `nginx:1.28.0-bookworm`. Please see the list of tags on docker hub
 # if you need to change NGX_VERSION.
@@ -8,7 +7,7 @@ ARG DEBIAN_RELEASE=bookworm
 # --- builder: build all examples
 FROM rust:slim-${DEBIAN_RELEASE} AS build
 ARG NGX_VERSION
-ARG NGX_CONFIGURE_ARGS
+ARG NGX_CONFIGURE_ARGS=
 WORKDIR /project
 RUN --mount=type=cache,target=/var/cache/apt <<EOF
     set -eux
```